src/Nethermind/Chains/sepolia.json

```
@@ -74,6 +74,7 @@
     "eip7251TransitionTimestamp": "0x67C7FD60",
     "eip7623TransitionTimestamp": "0x67C7FD60",
     "eip7702TransitionTimestamp": "0x67C7FD60",
+    "depositContractAddress": "0x4242424242424242424242424242424242424242",
```

```suggestion
    "depositContractAddress": "0x7f02c3e3c98b133055b8b348b2ac625669ed295d",
```

See: https://sepolia.etherscan.io/tx/0x025ecbf81a2f1220da6285d1701dc89fb5a956b62562ee922e1a9efd73eb4b14

---

ah, yeah, I was pushing the correct one